include/ultra64/mbi.h

```diff
@@ -58,15 +58,15 @@
  *
  **************************************************************************/
 
-#include "PR/gbi.h"
+#include "ultra64/gbi.h"
 
 /**************************************************************************
  *
  * Audio Binary Interface
  *
  **************************************************************************/
 
-#include "PR/abi.h"
+#include "ultra64/abi.h"
```

Comment:
Same concept here in audio. Only `audio/synthesis.c` uses `abi.h`, would I include `abi.h` or `mbi.h`?